plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/AttachContainerParams.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client.params;
+
+/**
+ * Arguments holder for {@code attachContainer} method of {@link org.eclipse.che.plugin.docker.client.DockerConnector}.
+ *
+ * @author Mykola Morhun
+ */
+public class AttachContainerParams {
+    /** id of container */
+    private String container;
+    /**
+     * if {@code true} gets 'live' stream from container.
+     * Note, that 'live' stream blocks until container is running.
+     */
+    private Boolean stream;
+
+    public AttachContainerParams withContainer(String container) {
+        this.container = container;
+        return this;
+    }
+
+    public AttachContainerParams withStream(Boolean stream) {
```

Comment:
why you need wrapper in args? wrapper is needed in filed to avoid setting default boolean, but what is the purpose to use it here?